docs/installation/instructions.rst

```diff
@@ -50,6 +50,15 @@ PyPI Installation
 
 7. Go through the :ref:`database-install`, :ref:`broker-install`, and `systemd-setup` sections
 
+.. note::
+
+    In place of using the systemd unit files provided in the `systemd-setup` section, you can run
+    the commands yourself inside of a shell. This is fine for development but not recommended in production.
+
+    $ /path/to/python/bin/celery worker -A pulpcore.tasking.celery_app:celery -n resource_manager@%%h -Q resource_manager -c 1 --events --umask 18
```

Comment:
Is there a way to escape this section?

Reply:
what do you mean exactly?

Reply:
<img width="957" alt="screen shot 2018-03-14 at 2 56 40 pm" src="https://user-images.githubusercontent.com/86290/37424873-f3169fc6-2797-11e8-8789-10cd93ae4a18.png">

Reply:
Updated, we'll see if this fixes it.